purchase_quick/views/product_view.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="product_product_tree_view" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field eval="20" name="priority"/>
+        <field name="arch" type="xml">
+            <tree string="Products" create="0" editable="top">
+                <field name="lst_price" readonly="1"/>
+                <field name="categ_id" readonly="1"/>
+                <field name="default_code" readonly="1"/>
+                <field name="name" readonly="1"/>
+                <field name="qty_to_purchase"/>
+                <field name="incoming_qty"/>
+                <field name="qty_available"/>
+                <button name="button_return_purchase" string="Return" type="object"/>
+            </tree>
+        </field>
+    </record>
+    <record id="product_search_form_view" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_search_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//filter[last()]" position="after">
```

Comment:
@PierrickBrun Here we use filter name also

Reply:
I don't see how this will improve the module. Isn't it more robust to use last() ? if the filter used is deleted by another module there won't be any problem with it but there will be if we use a name.

Reply:
IMHO I think last() is a good option because it resist to odoo upgrade even when name filter is renamed. Not blocking I imagine